src/components/v2/appDetails/k8Resource/nodeDetail/nodeDetail.util.ts

```diff
@@ -405,3 +407,13 @@ export const getTrimmedManifestData = (
 
     return returnAsString ? JSON.stringify(manifestData) : manifestData
 }
+
+export const getAppTypeCount = (appDetails) => {
```

Comment:
it should take appType and not whole appDetails as param
also the name should be relevant